### ISSUE TITLE
Adaptive Icon

### ISSUE THREAD

**Issue by splitt3r** [opacapp/opacclient#511]
Would be pretty nice to make the app icon adaptive. See https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive.html for further details.

Aaand thanks for the great app :muscle:

**Comment by johan12345** [comment]
I have added the adaptive icon, this is how it looks like now:

<img width="350" alt="unbenannt" src="https://user-images.githubusercontent.com/5310424/39362994-e78d4494-4a28-11e8-978f-78d09de1b926.PNG">

I'm still unsure about the background color - we could stay with white (with is what Android uses by default) - but this dark gray is also nice. @raphaelm and @splitt3r, what is your opinion on that?

**Comment by raphaelm** [comment]
I think white looks better, tbh, even if the contrast is a little low.

**Comment by johan12345** [comment]
Okay, I changed it to white now (see below).

<img width="162" alt="unbenannt" src="https://user-images.githubusercontent.com/5310424/39396397-b37962d0-4aed-11e8-90c3-34d4d0b82b0e.PNG">

I added the long diagonal shadow as seen above, but the drop shadow is difficult to implement for VectorDrawables - and I assume we don't want another big PNG (up to 324*324px) bloating the app download size. The AndroidAssetStudio tool doesn't generate any vector graphics (and no adaptive icons).
